src/olympia/amo/tests/__init__.py

```diff
@@ -497,11 +451,20 @@ def assertNoFormErrors(self, response):
                     if hasattr(v, 'non_form_errors'):
                         assert v.non_form_errors() == []
 
-    def assertLoginRedirects(self, *args, **kwargs):
-        return assertLoginRedirects(*args, **kwargs)
+    def assertLoginRedirects(self, response, to, status_code=302):
+        fxa_url = fxa_login_link(response, to)
+        self.assert3xx(
+            response=response,
+            expected_url=fxa_url,
+            status_code=status_code)
 
     def assert3xx(self, *args, **kwargs):
-        return assert3xx(*args, **kwargs)
+        """
+        Same as Django's assertRedirects but skips the final GET verification
+        step for performance reasons and backwards compatibility.
+        """
+        kwargs.setdefault('fetch_redirect_response', False)
+        return self.assertRedirects(*args, **kwargs)
```

Comment:
This already existed in Django 1.8, we can (and should) uplift this cleanup unrelated to Django 1.11.